packages/botonic-react/src/components/multichannel/multichannel-text.jsx

```diff
@@ -96,14 +103,19 @@ export const MultichannelText = props => {
     return generator
   }
 
-  const splitPostbackButtons = postbackButtons => {
+  const splitInWhatsappListButtons = postbackButtons => {
     const messages = []
-    for (let i = 0; i < postbackButtons.length; i += WHATSAPP_MAX_BUTTONS) {
-      messages.push(postbackButtons.slice(i, i + WHATSAPP_MAX_BUTTONS))
+    for (
+      let i = 0;
+      i < postbackButtons.length;
+      i += WHATSAPP_LIST_MAX_BUTTONS
+    ) {
+      messages.push(postbackButtons.slice(i, i + WHATSAPP_LIST_MAX_BUTTONS))
     }
     return messages
   }
 
+  // START WHATSAPP LOGIC
```

Comment:
Why is this comment added here?

Reply:
It is not necessary but it helps to see where the logic begins. I would like to make a refactor to extract this into a MultichannelWhatsappText component.